### ISSUE TITLE
Browser spelling check at Markdown unfortunately does not work - what do

### ISSUE THREAD

**Issue by AlexL777** [Fred89/flatboard#9]
Hello Developers,

I have a desire for the flatboard and I would like somehow to realize the browser spelling check. Currently it does not work with Google Chrome in me and in Markdown.

Is there a flatboard plugin or can you possibly develop under certain circumstances?

Then I wait for your feedback.

Thanks in advance.

**Comment by Fred89** [comment]
Hi Alex,
I update Flatboard with the last SimpleMDE plugin, try it please.

**Comment by AlexL777** [comment]
Hi @Fred89 , unfortunately, the SpellCheck does not work in my browser Microsoft Edge. I have only recently switched to this browser. In which browser does it work with you?

**Comment by AlexL777** [comment]
Hello Developers,

I found in the Markdown Editor file today that the SpellChecker was "false". Now I have set it to "true", but I do not understand how the SpellCheck in the frontend function in the forum.

I'll show you a screenshot and so it looks with me when posting in the forum.

I write in German and normally correct German words should not be highlighted in color, but they do.

What am I doing wrong?

Thanks in advance.

<img width="350" alt="flatboard-spellchecker-not-worked-issue" src="https://cloud.githubusercontent.com/assets/7885632/24577181/4ba1408c-16c9-11e7-8bce-f92815df926b.png">
